src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.java

```diff
@@ -545,11 +544,12 @@ private boolean hasEmptyLine(int startLine, int endLine) {
      */
     private boolean hasEmptyLineBefore(DetailAST token) {
         final int lineNo = token.getLineNo();
+        boolean result = false;
         if (lineNo == 1) {
-            return false;
+            return result;
```

Comment:
You extracted the variable result, assigned it to false then returned it so why did you create it in the first place?